docs/conf/portland/2018/speaking-tips.rst

```diff
@@ -0,0 +1,7 @@
+:template: {{year}}/generic.html
+:banner: _static/2018/assets/headers/writing-day.png
+
+Speaking Tips
+=================
+
+.. include:: /include/conf/speaking-tips.rst
```

Comment:
This data does feel conference specific (eg. resolution, length, etc.) -- is a generic include the best approach?